<?xml version="1.0"?>
<launch>

	<param name="rho"        type="double" value="1.0" />
	<param name="theta"      type="double" value="0.017452778" />
    <param name="threshold"  type="double" value="100.0" />
    <param name="minLineLen" type="double" value="100.0" />
    <param name="maxLineGap" type="double" value="10.0" />

    <!-- <param name="minCan"     type="double" value="250" />
    <param name="maxCan"     type="double" value="255" /> -->

    <param name="imgTopic"   type="str" value="/camera/color/image_raw"/>

    <node pkg="vision_matrice100" type="image_filter_real_node" name="image_filter_real_node" output="screen"/>

</launch>

